src/commands/quickstart.ts

```diff
@@ -196,20 +199,21 @@ export default class Quickstart extends Command {
       logCb?: LogCallback;
       warnCb?: LogCallback;
     }
-  ): Promise<string | undefined> {
+  ): Promise<void> {
     const response: { value: string } = await inquirer.prompt({
       name: 'value',
       message: `Enter ${name} of ${authType} security for "${provider}" This value will be stored locally in .env file.`,
       type: 'input',
     });
-    if (process.env[envVariableName]) {
+    const variableName = envVariableName.startsWith('$')
+      ? envVariableName.substring(1)
+      : envVariableName;
```

Comment:
When the `$` is not present then the value in super.json is the actual secret, so I think this should error/warn here as we don't want to end up using the secret as the env variable key :D

Reply:
Well the value is set by configure logic so it shouldn't be the actual logic 🤷 But we can warn/err :)

Reply:
I'd warn and skip the value, as it's either a bug in configure or some now-unforseen configuration that might be implemented later.

Reply:
I will just warn the user but won't throw or exit command.